### PR TITLE
Translation refactor

### PR DESCRIPTION
**Pull request by billybonks**
Works on https://github.com/HospitalRun/hospitalrun-frontend/issues/547
This updates

- pricing
- imaging
- diagnosis
- labs

A few different modules ported over.

Text stays relatively the same, there is a pull request/s in the works for:

- medication.
- invoices.
- inventory.
- procedure.

the last remaining module is

- patients.

The main goal here it easy to know where translations belong, i will be typing out some documentation upon completion of the port.

This is a breaking change for locales i am not sure if they are deployed somewhere if you would like to make a locale refactor branch to merge into i will be happy to do that. 

i will implement transfix as soon as the refactor is completion.

thanks to @gabrielle-ong and @RenchChua

### CODE REVIEW

File: app/locales/en/translations.js [hunk 1]
```diff
@@ -1263,6 +1249,23 @@ export default {
...
+        requestedBy: 'Requested By',
+        radiologist: 'Radiologists',
+        patient: 'Patient',
+        imagingType: 'Imaging Type',
+        result: 'Result',
+        notes: 'Notes',
+        visit: 'Visit',
+        status: 'Status'
+      },
+      placeholders: {
```
Comment (jkleinsc) [inline app/locales/en/translations.js]: @billybonks I'm not sure about calling this a placeholder.  It is a valid option to select, not a temporary placeholder value.  When you select/leave this option a new visit is created, so there is action firing based on its selection/non selection.

File: app/locales/en/translations.js [hunk 2]
```diff
@@ -1278,6 +1278,27 @@ export default {
         appointmentDate: 'Date'
       }
     },
+    labs: {
```
Comment (codeNgJon) [inline app/locales/en/translations.js]: Hey, I'm interested in helping out with the translations. Wondering why `labs` is pluralized when `pricing` is singular?

Comment (codeNgJon) [inline app/locales/en/translations.js]: jk. just saw the fix in the later commit

**Comment by jkleinsc** [comment]
@billybonks I have been waiting to merge this in until the placeholder issue I brought up is resolved.  I'm fine waiting on it, but just thought I would ping you to make sure we are on the same page on this one.

**Comment by tangollama** [comment]
@billybonks bump. :)

**Comment by tangollama** [comment]
@jkleinsc I *think* this is ready to go, but I'll leave that to you.

**Comment by jkleinsc** [comment]
@tangollama @billybonks I had asked that the "placeholder" label be changed.  That I can see, it has not been changed.

**Comment by tangollama** [comment]
@billybonks closing this PR for now. Feel free to re-open when/if it makes sense.

**Pull request closed without merging**